webpack/config.js

```diff
@@ -97,30 +97,50 @@ var plugins = [
 var vendor_modules = [
   "autosize",
   "babel-polyfill",
+  "buffer-crc32",
   "classnames",
   "codemirror",
```

Comment:
It'd be amazing if the codemirror stuff was a dynamic import!

Reply:
I actually looked into it, it's like only 60kb gzipped, and our total vendor size is 390kb gzipped, not too shabby!

Relay, react and something called `core-js` are the biggest players.

Reply:
Also React 16 is much smaller and has dropped the vendor bundle by about 20kb